.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -0,0 +1,36 @@
+---
+name: New Release Template
+about: New Release checklist and template
+title: 'chore: tag {version}'
+labels: ''
+assignees: ''
+
+---
+
+# New Release Checklist
+
+ - [ ] Switch to the `main` branch.
+ - [ ] `git pull` to ensure the local copy is completely up-to-date.
+ - [ ] `git diff origin/main` to ensure there are no local staged or uncommitted changes.
```

Comment:
I suppose contributors who use a fork would know that they would need to replace `origin` with the remote name of the upstream repo that they chose. 🤔